projects/phone-field/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@skyux/phone-field",
+  "version": "5.0.0-beta.0",
+  "peerDependencies": {
+    "@angular/animations": "^12.2.0",
+    "@angular/common": "^12.2.0",
+    "@angular/core": "^12.2.0",
+    "@angular/forms": "^12.2.0",
+    "@skyux/forms": "^5.0.0-beta.0",
+    "@skyux/i18n": "^5.0.0-beta.0",
+    "@skyux/indicators": "^5.0.0-beta.0",
+    "@skyux/lookup": "^5.0.0-beta.0",
+    "@skyux/theme": "^5.0.0-beta.0"
```

Comment:
The SKY UX packages need updated to match the versions in the root `package.json`.

Reply:
Fixed